### DISCUSSION TITLE
Close : Custom query: please provide a non-destructive SQL statement.

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/biblioteq#211] [Q&A] [answered]
I tried to copy a field into another one

INSERT INTO book (call_number) SELECT deweynumber FROM book

BiblioteQ answers me a message  "Please provide a non-destructive SQL statement."

what am I doing wrong?

**Comment by textbrowser** [comment]
That's a destructive statement. Only SELECT queries are allowed.

**Reply by meteos77** [reply]
ok

**Reply by meteos77** [reply]
insert is destructive

**Comment by textbrowser** [comment]
So are updates. Deletes.

**Comment by textbrowser** [comment]
Yes, non-destructive queries do not modify the state of the database. There are some exceptions. For example, sequences.

**Reply by meteos77** [reply]
some exceptions ?

**Comment by textbrowser** [comment, marked as answer]
I will add a little more information to the text.
